experiments/ppo_gridnet.py

```diff
@@ -236,6 +237,66 @@ def get_value(self, x):
         return self.critic(self.encoder(x))
 
 
+def run_evaluation(model_path: str, output_path: str):
+    args = [
+        "python",
+        "league.py",
+        "--evals",
+        model_path,
+        "--update-db",
+        "false",
+        "--cuda",
+        "false",
+        "--output-path",
+        output_path
+    ]
+    fd = subprocess.Popen(args)
+    print(f"Evaluating {model_path}")
+    return_code = fd.wait()
+    assert return_code == 0
+    return (output_path, model_path)
+
+
+class TrueskillWriter:
+
+    def __init__(self, prod_mode, writer, league_path: str, league_step_path: str):
+        self._prod_mode = prod_mode
+        self._writer = writer
```

Comment:
Is there a reason to make these private variables?

Reply:
Just a habit, I can make them public

Reply:
That will be great! Should be easier to debug sometimes :)

Reply:
I changed them to be public